styles.css

```diff
@@ -648,16 +648,129 @@ li.language {
   color: #fff;
 }
 
+.shape {
+  position: absolute;
+  bottom: 8px;
+  left: 45%;
+  width: 134px;
+  height: 15px;
+  border-radius: 10px;
+  background-color: #0d151e;
+}
+
 .logo-nav .nav-bar li:hover {
   background-color: #396df2;
   border-radius: 8px;
 }
 
 @media only screen and (max-width: 768px) {
-  .nav-bar {
+  nav {
     display: none;
   }
 
+  nav.menu-btn {
+    display: block;
+    position: absolute;
+    margin-left: auto;
+    margin-right: auto;
+    top: 0;
+    width: 375px;
+    height: 792px;
+    z-index: 800;
+    background: #6070ff;
+    padding: 1em;
+    color: #fff;
+    opacity: 0.95;
+  }
```

Comment:
- [ ] [REQUIRED] Good job so far 👍  If you noticed that there is a `space` on both sides of the mobile menu and when `scrolling` the mobile menu, the mobile menu elements are missing and the body elements are starting to appear, please fix these problems. it's easy for you and I trust you to do it 😃 

![image](https://user-images.githubusercontent.com/63915024/196482092-34d72c6c-7bf9-440c-9130-5c034b081005.png)